ocp_resources/resource_quota.py

```diff
@@ -0,0 +1,26 @@
+# API reference: https://docs.openshift.com/container-platform/4.13/applications/quotas/quotas-setting-per-project.html
+
+from ocp_resources.resource import NamespacedResource
+
+
+class ResourceQuota(NamespacedResource):
+    api_version = NamespacedResource.ApiVersion.V1
+
+    def __init__(
+        self,
+        hard=None,
+        **kwargs,
+    ):
+        """
+        Create ResourceQuota object.
+
+        Args:
+            hard (dict): set of desired hard limits
+        """
+        super().__init__(**kwargs)
+        self.hard = hard
+
+    def to_dict(self):
+        super().to_dict()
+        if not self.yaml_file:
+            self.res.setdefault("spec", {})["hard"] = self.hard
```

Comment:
ditto, required?

Reply:
same here - it can be created with empty `hard`

Reply:
ack